src/graph/GoExecutor.cpp

```diff
@@ -461,6 +466,11 @@ Status GoExecutor::checkNeededProps() {
 
 
 Status GoExecutor::setupStarts() {
+    SCOPE_EXIT {
+        // unique the start vid
```

Comment:
This is not a bug.

Reply:
Emm. It's not related to this issue, so I will revert from this PR, but I'm not sure about it's feature or bug .

Reply:
This had fixed in #2292 and 2.0